test/shared/src/test/scala/monocle/IsoSpec.scala

```diff
@@ -123,21 +123,20 @@ class IsoSpec extends MonocleSuite {
     iso.modify(_ + 1)(IntWrapper(0)) shouldEqual IntWrapper(1)
   }
 
-  test("GenIso nullary equality") {
-    GenIso.unit[Nullary] shouldEqual _nullary
-  }
-
-  test("GenIso unary equality") {
-    GenIso[Unary, Int] shouldEqual _unary
-  }
+  // test("GenIso nullary equality") {
```

Comment:
Here are the eq instance: https://github.com/julien-truffaut/Monocle/blob/cats/test/shared/src/test/scala/monocle/TestInstances.scala#L50-L54